vars/sharedInfrastructurePipeline.groovy

```diff
@@ -13,6 +13,22 @@ def call(String product, String environment, String subscription, boolean planOn
 
 @Deprecated
 def call(String product, String environment, String subscription, boolean planOnly, String deploymentTarget) {
+  echo '''
+================================================================================
```

Comment:
what do you think about sending a slack message on every build that's its deprecated? or is that overkill?
It's much more likely to be noticed then

Reply:
Yeah - that's why it's being deprecated 😂